ansible/README.md

```diff
@@ -1,5 +1,24 @@
 # Ansible playbooks to download and install dependencies for OpenJDK on various platforms
 
+# Quickstart Guide
+
+To test the ansible scripts, you'll need to install the following programs.
+
+For macOS:
+
+1) Install Homebrew 2.1.7 or later
```

Comment:
nit: can you change your numbers to be `1.` rather than `1)` please? It's the standard for markdown.